sequencer/src/api/endpoints.rs

```diff
@@ -46,6 +46,42 @@ pub struct NamespaceProofQueryData {
     pub transactions: Vec<Transaction>,
 }
 
+pub(super) fn get_balance<State, Ver>() -> Result<Api<State, merklized_state::Error, Ver>>
+where
+    State: 'static + Send + Sync + ReadState,
+    Ver: 'static + StaticVersionType,
+    <State as ReadState>::State: Send
+        + Sync
+        + MerklizedStateDataSource<SeqTypes, FeeMerkleTree, 256>
```

Comment:
Define a constant for the 256 arity?

Reply:
I cannot compile use FeeMerkleTree::ARITY

error[E0747]: type provided when a constant was expected

Reply:
I think Mathis is right, we need to define the constant.
even in the type alias FeeMerkleTee we use 256 as the type parameter for ARITY